ci/tests/rp/test.sh

```diff
@@ -8,14 +19,12 @@ echo "--- smoke test $ret"
 
 echo '--- unit test'
 pytest -vvv tests/unit_tests
-test "$ret" = 0 && ret=$?
+test "$ret" = 0 || ret=$?
```

Comment:
I don't understand that change: if a previous test suceeded (`ret=0`) and this one failed (`$?=1`), then the new version will ignore that failure:
```sh
$ ret=0
$ test "$ret" = 0 || ret=1
$ echo $ret
0
```

Reply:
right.. should we just remove `test "$ret" = 0 ||`? and only get return value to print it out

Reply:
I think this should be left as is.  The rationale is that we want to run all tests, but if any one of them returns a non-zero exit code, then that will be used as the final result.  The current code does just that. (`exit $ret`).

Reply:
Ok, got it! Thanks!